compiler/passes/convert-uast.cpp

```diff
@@ -1981,6 +2070,28 @@ struct Converter {
     return ret;
   }
 
+  CallExpr* convertModuleDotCall(const uast::FnCall* node) {
+    ID moduleId, targetId;
+    types::QualifiedType targetType;
+    auto dot = node->calledExpression()->toDot();
+    if (!dot || !isDotOnModule(dot, moduleId, targetId, targetType)) return nullptr;
+
+    // Don't know how to convert fabricated IDs yet.
+    if (targetId.isFabricatedId()) return nullptr;
```

Comment:
Should this assert instead / in addition?

Reply:
I can make it assert if the fabricated ID is not extern, but the production scope resolver should be handling these for now (so no assertion, just skip)